Assets/Scripts/Networking/GameConnectionSystem.cs

```diff
@@ -0,0 +1,44 @@
+using Unity.Entities;
+using Unity.NetCode;
+using Unity.Networking.Transport;
+using UnityEngine;
+
+public struct InitGameComponent : IComponentData {}
```

Comment:
I would recommend renaming this to be more descriptive and maybe moving it inside its parent class.